omod/src/main/resources/config.xml

```diff
@@ -26,14 +26,18 @@
 		${project.parent.description}
 	</description>
 	<updateURL>https://modules.openmrs.org/modules/download/${project.parent.artifactId}/update.rdf</updateURL>
-	<require_version>2.0.5 - 2.0.*, 1.11.3, 1.9.10 - 1.9.*</require_version>
+	<require_version>2.2.*, 2.0.5 - 2.0.*, 1.11.3, 1.9.10 - 1.9.*</require_version>
```

Comment:
Since platform 2.3 is getting released, can we add support for it too? And platform 2.4 which is the master branch. This way people can tests the module on master.